src/test/resources/features/contracts.feature

```diff
@@ -0,0 +1,9 @@
+Feature: EO-06- Contracts
+
+  Scenario: EO-06 Verificar si se puede crear un contrato SIN los campos requeridos
+    Given 'SSI Application' page is loaded
+    And set Admin credentials on 'Login' page
+    And click 'Estructura Organizacional' menu button on 'Navigation Bar' top menu
+    And click 'Contratos' option on 'Estructura Organizacional' submenu
+    When 'Contracts Form' page is fill with some contract data
```

Comment:
Let's change it to "And" and i think It would be 
'Contracts Form' page is filled with contract data